src/subscription_manager/utils.py

```diff
@@ -229,6 +197,32 @@ def get_current_owner(uep=None, identity=None):
     return cache.read_data(uep, identity)
 
 
+def get_content_access_mode(uep=None, identity=None, owner=None):
```

Comment:
This method is unused and IMO should be removed along with the other InMemory* cache bits.

Reply:
I agree, am removing.